components/pages/privacy/resourceAccess/resourceAccessDrawer/index.jsx

```diff
@@ -256,7 +255,7 @@ export default function ResourceAccessDrawer({
           >
             Remove Access to {resourceName}
           </button>
-          <ConfirmationDialog />
+          {/* <ConfirmationDialog /> */}
```

Comment:
Why is this commented out as opposed to removed?

Reply:
🙈  removed it